src/satisfiers.jl

```diff
@@ -0,0 +1,99 @@
+
```

Comment:
Why is this all showing as new conetent on github?
Isn;t that already there?

Reply:
I renamed the file to fix a typo (missing an "s"). I'll see what git magic I can do to get it to recognize it as a rename.

Reply:
Forgot the `git rm` on my first push